### PR TITLE
Create users table

### PR DESCRIPTION
**Pull request by gabrielrom**
# Commits
- :art: refactor: add data annotations attr in user
- :sparkles: feat: add table users on dbcontext
- :sparkles: feat: create the table users on database

# Columns created:
 - id: primary key
 - name: the name of user, type is text
 - email: the email of user, type is text
 - password: the password of a user, the type is text too
 - avatar_url: the url of image user, type is text and can be nullable
 - account_id: the id of account user, can be nullable too

### CODE REVIEW

File: src/modules/users/entities/User.cs [hunk 1]
```diff
@@ -1,10 +1,36 @@
...
+    [Column]
+    [Required]
     public string password { get; set; }
+
+    [Column]
     public string avatar_url { get; set; }
+
+    [Column]
     public string account_id { get; set; }
+
```
Comment (RichflairDrip) [inline src/modules/users/entities/User.cs:29]: The table of user needs two columns called created_at and updated_at.

**Review by RichflairDrip — CHANGES_REQUESTED** [review]
The table just need some refactor to add columns created_at and updated_at

File: src/modules/users/entities/User.cs (hunk 1 above)
Comment (gabrielrom) [reply src/modules/users/entities/User.cs:29]: Hey @RichflairDrip, the changes have been made, can you take a look please!

**Review by RichflairDrip — COMMENTED** [review]
Good job @gabrielrom !

**Pull request merged**